lib/main.dart

```diff
@@ -1,67 +1,315 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-
+import 'package:flutter/services.dart';
 void main() {
-  runApp(const MyApp());
+  WidgetsFlutterBinding.ensureInitialized();
+  SystemChrome.setPreferredOrientations([DeviceOrientation.landscapeLeft])
+      .then((_) {
+  runApp(MyApp());
+  });
 }
 
+var marginDivider = 1;
+double orientationWidth = 1;
+double orientationHeight = 1;
+
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
-
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const HomePage(),
     );
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({Key? key, required this.title}) : super(key: key);
-  final String title;
+class HomePage extends StatefulWidget {
```

Comment:
You app (`HomePage`) is currently implemented as `StatefulWidget`, with all your UI logic inside the state. 

I haven't explained the difference between `StatefulWidget`, `StatelessWidget` and `State<T>` yet, but here, you can make your entire `HomePage` a `StatelessWidget` and get rid of your state (just move all code to `build` method of `HomePage`)